src/main/java/com/dlsc/jfxcentral/data/DataRepository2.java

```diff
@@ -1,26 +1,7 @@
 package com.dlsc.jfxcentral.data;
 
-import com.dlsc.jfxcentral.data.model.Blog;
```

Comment:
We use fully qualified imports. Please change or we constantly have tons of deltas without any reason.

Reply:
ok, done, my IDE decided to make this change...